pkg/registry/remote.go

```diff
@@ -44,7 +46,16 @@ func readRemoteImage(n image.Name) (v1.Image, error) {
 		return nil, err
 	}
 
-	return remote.Image(ref, remote.WithAuth(auth))
+	img, err := daemon.Image(ref)
+	if err != nil {
+		var remoteErr error
+		img, remoteErr = remote.Image(ref, remote.WithAuth(auth))
+		if  remoteErr != nil {
```

Comment:
tiny formatting issue here
```suggestion
		if remoteErr != nil {
```